src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig

```diff
@@ -1 +1,24 @@
-Welcome!
+<div class="card">
+    <div class="card-body">
+        <div class="jumbotron p-1 bg-white mb-0">
+            <h1 class="display-4">Welcome</h1>
+            <p class="lead">in your Chameleon System open source backend</p>
```

Comment:
```suggestion
            <p class="lead">in your Chameleon System backend</p>
```

("open source" implies there is a non-open-source version, but there isn't)

Reply:
removed

Reply:
Hmmm in fact, this was the reason why i added it. There ARE non open source versions. All versions before 6.2.

Reply:
But those are legacy versions that got replaced by 6.2 - we don't need to distinguish this release from anything outdated and superseded. I'd agree to your proposal if there was a "community edition" and e.g. a "commercial edition" with different functionality.